2025-04-15 09:55:13,565 - INFO - Using CPU-build of llama.cpp
2025-04-15 09:55:13,590 - INFO - Benchmarking model SmolLM-135M.Q4_K_M.gguf ...
2025-04-15 09:55:13,594 - DEBUG - Downloading model SmolLM-135M.Q4_K_M.gguf from https://huggingface.co/QuantFactory/SmolLM-135M-GGUF/resolve/main/SmolLM-135M.Q4_K_M.gguf
2025-04-15 09:55:14,025 - DEBUG - Downloaded model SmolLM-135M.Q4_K_M.gguf (100.57 MB) in 0.43 sec (233.89 MB/s)
2025-04-15 09:55:14,027 - DEBUG - Downloading model qwen1_5-0_5b-chat-q4_k_m.gguf from https://huggingface.co/Qwen/Qwen1.5-0.5B-Chat-GGUF/resolve/main/qwen1_5-0_5b-chat-q4_k_m.gguf
2025-04-15 09:55:14,029 - DEBUG - Model SmolLM-135M.Q4_K_M.gguf found at /models/SmolLM-135M.Q4_K_M.gguf (0.10 GB)
2025-04-15 09:55:14,029 - DEBUG - Using ngl 0 for model SmolLM-135M.Q4_K_M.gguf
2025-04-15 09:55:14,029 - DEBUG - Benchmarking prompt processing with 16 tokens for max 41 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
Process Process-2:
Traceback (most recent call last):
  File "/usr/lib/python3.10/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/usr/lib/python3.10/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/benchmark.py", line 163, in download_models
    urlretrieve(model_url, temp_path)
  File "/usr/lib/python3.10/urllib/request.py", line 241, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
  File "/usr/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 500: Internal Server Error
2025-04-15 09:55:14,193 - DEBUG - Benchmarking prompt processing with 128 tokens for max 65 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:55:14,508 - DEBUG - Benchmarking prompt processing with 512 tokens for max 104 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:55:15,624 - DEBUG - Benchmarking prompt processing with 1024 tokens for max 104 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:55:17,691 - DEBUG - Benchmarking prompt processing with 4096 tokens for max 83 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:55:31,627 - DEBUG - Benchmarking prompt processing with 16384 tokens for max 83 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:56:54,647 - ERROR - Error: Command '['./llama-bench', '-t', '36', '-sm', 'layer', '-fa', '1', '-ub', '512', '-b', '2048', '-o', 'jsonl', '-m', '/models/SmolLM-135M.Q4_K_M.gguf', '-ngl', '0', '-p', '16384', '-n', '0']' timed out after 82.99997910400003 seconds
2025-04-15 09:56:54,647 - DEBUG - Benchmarking text generation with 16 tokens for max 81 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:56:55,012 - DEBUG - Benchmarking text generation with 128 tokens for max 129 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:56:57,230 - DEBUG - Benchmarking text generation with 512 tokens for max 104 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:57:06,409 - DEBUG - Benchmarking text generation with 1024 tokens for max 104 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:57:24,852 - DEBUG - Benchmarking text generation with 4096 tokens for max 83 sec
load_backend: loaded CPU backend from ./libggml-cpu-sapphirerapids.so
2025-04-15 09:58:47,861 - ERROR - Error: Command '['./llama-bench', '-t', '36', '-sm', 'layer', '-fa', '1', '-ub', '512', '-b', '2048', '-o', 'jsonl', '-m', '/models/SmolLM-135M.Q4_K_M.gguf', '-ngl', '0', '-n', '4096', '-p', '0']' timed out after 82.9999846400001 seconds
2025-04-15 09:58:47,862 - INFO - Benchmarking model qwen1_5-0_5b-chat-q4_k_m.gguf ...
Traceback (most recent call last):
  File "/benchmark.py", line 292, in <module>
    target_file_size = get_model_url_size(model_url)
  File "/benchmark.py", line 140, in get_model_url_size
    with urlopen(Request(model_url, method="HEAD")) as response:
  File "/usr/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 500: Internal Server Error
